tests/api_connexion/endpoints/test_task_instance_endpoint.py

```diff
@@ -376,6 +376,16 @@ def test_should_return_404_for_mapped_endpoint(self, session):
             )
             assert response.status_code == 404
 
+    def test_should_return_404_for_list_mapped_endpoint(self, session):
+        self.create_task_instances(session)
+        for index in ['0', '1', '2']:
+            response = self.client.get(
+                "/api/v1/dags/example_python_operator/dagRuns/TEST_DAG_RUN_ID/"
+                "taskInstances/print_the_context/listMapped",
+                environ_overrides={"REMOTE_USER": "test"},
+            )
+            assert response.status_code == 404
```

Comment:
I don't think this for loop is needed/does anything now
```suggestion
        response = self.client.get(
            "/api/v1/dags/example_python_operator/dagRuns/TEST_DAG_RUN_ID/"
            "taskInstances/print_the_context/listMapped",
            environ_overrides={"REMOTE_USER": "test"},
        )
        assert response.status_code == 404
```

Reply:
Yep, that was thoughtless copy+paste on my part.